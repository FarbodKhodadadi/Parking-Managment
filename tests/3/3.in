request_spot Honda_Fit
assign_spot 101
assign_spot 301
request_spot Peraid
request_spot Samand
request_spot Peraid
assign_spot 245
request_spot Ford_Expedition
assign_spot 321
request_spot Pershia
checkout 101
checkout 301
assign_spot 255
checkout 255
checkout 321
