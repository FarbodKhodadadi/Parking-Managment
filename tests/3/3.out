101: normal 250 50
255: CCTV 330 110
Spot 101 is occupied now.
Spot 301 is occupied now.
245: normal 300 70
201: CCTV 430 140
321: normal 350 80
443: covered 400 110
245: normal 300 70
Spot 245 is occupied now.
201: CCTV 430 140
321: normal 350 80
443: covered 400 110
Spot 321 is occupied now.
201: CCTV 430 140
443: covered 400 110
Spot 101 is free now.
Total cost: 250
Spot 301 is free now.
Total cost: 350
Spot 255 is occupied now.
Spot 255 is free now.
Total cost: 330
Spot 321 is free now.
Total cost: 350
